components/style_traits/values.rs

```diff
@@ -38,6 +38,20 @@ impl<'a, T> ToCss for &'a T where T: ToCss + ?Sized {
     }
 }
 
+impl ToCss for str {
```

Comment:
Please add a comment in the ToCss definition pointing to this, since it may be easy to misuse and end up serialising other stuff in a way it was not intended to.